tool/gravity/cli/commands.go

```diff
@@ -76,6 +76,8 @@ type Application struct {
 	PlanExecuteCmd PlanExecuteCmd
 	// PlanRollbackCmd rolls back a phase of an active operation
 	PlanRollbackCmd PlanRollbackCmd
+	// ResumeCmd resumes active operation
+	ResumeCmd ResumeCmd
```

Comment:
I also tentatively added `gravity resume` as it was discussed and had also to do with this PR.